--- a/quazip/QuaZipConfig.cmake.in
+++ b/quazip/QuaZipConfig.cmake.in
@@ -11,7 +11,7 @@ elseif(@QUAZIP_QT_MAJOR_VERSION@ EQUAL 5
 elseif(@QUAZIP_QT_MAJOR_VERSION@ EQUAL 4)
 	find_dependency(Qt4 4.5.0 REQUIRED COMPONENTS QtCore)
 else()
-    message(FATAL_ERROR "Qt version QUAZIP_QT_MAJOR_VERSION=@QUAZIP_QT_MAJOR_VERSION@ is unsupported")
+    message(FATAL_ERROR "Qt version QUAZIP_QT_MAJOR_VERSION=${QUAZIP_QT_MAJOR_VERSION} is unsupported")
 endif()
 
 if(@QUAZIP_BZIP2@)
@@ -24,6 +24,6 @@ else()
     find_dependency(ZLIB REQUIRED)
 endif()
 
-set_target_properties(QuaZip::QuaZip PROPERTIES IMPORTED_GLOBAL TRUE)
+#set_target_properties(QuaZip::QuaZip PROPERTIES IMPORTED_GLOBAL TRUE)
 
 check_required_components(@QUAZIP_PACKAGE_NAME@)

